pages/common/fly.md

```diff
@@ -0,0 +1,35 @@
+# fly
+
+> Cli tool for concourse-ci.
+
+- Authenticate with and save concourse target:
+
+`fly --target {{example}} login --team-name {{my-team}} -c {{https://ci.example.com}}`
+
+- List targets:
+
+`fly targets`
+
+- List pipelines:
+
+`fly -t {{example}} pipelines`
+
+- Configure pipeline:
```

Comment:
What is meant by "configure" here? Does it mean that the configuration file is set? I've no experience with this command, but perhaps it could be reworded to help beginners.